docs/src/models/atmodel.rst

```diff
@@ -0,0 +1,9 @@
+:mod:`models.atmodel` -- Author-topic models
+======================================================
+
+.. automodule:: gensim.models.atmodel
+    :synopsis: Author-topic models
+    :members:
```

Comment:
check html is generated in https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#documentation